pages/blog/conference-2022.md

```diff
@@ -0,0 +1,185 @@
+---
+title: "Conference 2022 Was Amazing!"
+date: 2022-12-19T10:00:00+01:00
+type: Conference
+tags:
+  - Conference
+cover: /img/posts/conference-2022.webp
+featured: true
+authors:
+  - name: Azeez Elegbede
+    photo: /img/avatars/ace.webp
+    link: https://twitter.com/_acebuild
+    byline: Preacher at AsyncAPI
```

Comment:
```suggestion
    byline: AsyncAPI DevRel
```

`Preacher` is about church stuff, not OSS 😄

Reply:
Don't mind me trying to give myself a title 😄.